src/org/labkey/trialshare/TrialShareController.java

```diff
@@ -584,8 +584,11 @@ public boolean shouldExport(TrialShareExportForm form)
 
     public static class TrialShareExportTest
     {
+        // ignore these export types because they won't appear in the folder export UI
+        private static final Set<String> _ignoredFolderWriters = Set.of("Notebooks", "LabBooks");
+
         private final Collection<FolderWriter> folderWriters = FolderSerializationRegistry.get().getRegisteredFolderWriters().stream()
-                .filter(fw -> !"Notebooks".equals(fw.getDataType()))
+                .filter(fw -> !_ignoredFolderWriters.contains(fw.getDataType()))
```

Comment:
Could we filter out all hidden data types to avoid maintaining this list in perpetuity?
```suggestion
                .filter(fw -> fw.show(ContainerManager.getSharedContainer()))
```
Or maybe filter them out in `getRegisteredDataTypes`